examples/example_frbc_rm.py

```diff
@@ -146,7 +146,7 @@ def deactivate(self, conn: S2Connection) -> None:
 
 
 s2_conn = S2Connection(
-    url="ws://localhost:8001/backend/rm/s2python-frbc/cem/dummy_model/ws",
+    url="ws://localhost:8080/backend/rm/s2python-frbc/cem/dummy_model/ws",
```

Comment:
Is this a better example port?

Reply:
I did it to match the web-socket example from the fm-client